etc/kubernetes/manifests/base/apps/cos-fleetshard-operator-camel/app-config/application.properties

```diff
@@ -5,5 +5,5 @@ quarkus.log.category."org.bf2.cos.fleetshard.sync".level = DEBUG
 quarkus.log.category."org.bf2.cos.fleetshard.client".level = DEBUG
 quarkus.log.category."io.fabric8.kubernetes.client.internal.VersionUsageUtils".level = ERROR
 
-cos.operator.camel.label-selection.enabled = false
+cos.operator.camel.label-selection.enabled = true
```

Comment:
Is the value `cos-fleetshard-operator-camel` configurable? Should we make it in a future PR? Just a question/suggestion. LGTM.